rest-api/tools/export_tables.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+# Exports database tables to CSV files in GCS.
+
+USAGE="tools/export_tables.sh --project <PROJECT> --account <ACCOUNT> --bucket <BUCKET> --directory <DIRECTORY> --database <DATABASE> --tables <TABLES> [--creds_account <ACCOUNT>]"
+while true; do
+  case "$1" in
+    --account) ACCOUNT=$2; shift 2;;
+    --creds_account) CREDS_ACCOUNT=$2; shift 2;;
+    --project) PROJECT=$2; shift 2;;
+    --bucket) BUCKET=$2; shift 2;;
+    --directory) DIRECTORY=$2; shift 2;;
+    --database) DATABASE=$2; shift 2;;
+    --tables) TABLES=$2; shift 2;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+if [ -z "${ACCOUNT}" ] || [ -z "${PROJECT}" ] || [ -z "${BUCKET}" ] || [ -z "${DATABASE}" ] \
+  || [ -z "${TABLES}" ] || [ -z "${DIRECTORY}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${CREDS_ACCOUNT}" ]
+then
+  CREDS_ACCOUNT="${ACCOUNT}"
+fi
+
+source tools/auth_setup.sh
+SQL_SERVICE_ACCOUNT=`gcloud sql instances describe --project ${PROJECT} --account ${ACCOUNT} \
```

Comment:
This looks important -- bubble it up to the top-level docs? We're taking ownership of the specified bucket?

Reply:
I changed the logic to remove the service account from the ACL when the script finishes. Added a comment at the top.

Reply:
Thanks!